net/lp_rpc.proto

```diff
@@ -301,6 +301,7 @@ message VideoProfile {
     CHROMA_444 = 2;
   }
   ChromaSubsampling chromaFormat = 27;
+  uint32 crf = 28;
```

Comment:
Is this the default? I'd suggest setting this lower since it translates to a pretty high CQ.

Reply:
OK i see now after reading your comment in the other PR.

Reply:
This is not a value, just the field number in the probuf. So, in short, in just needs to be the next number after the line above.